dashboard/pages/Epic.py

```diff
@@ -1,18 +1,249 @@
-'''A script that creates the epic page for the dashboard.'''
+'''A script that creates the Steam page for the dashboard.'''
 
 from os import environ as ENV
+from datetime import datetime, timedelta
 
+import pandas as pd
+import altair as alt
 import streamlit as st
+from dotenv import load_dotenv
+
+from psycopg2 import connect
+from psycopg2.extras import RealDictCursor
+from psycopg2.extensions import connection
+
+
+def get_db_connection(config) -> connection:
```

Comment:
could these functions be imported from another file? rather then repeating them again?

Reply:
Absolutely can do for some but not all, will have a look 👍